internal/services/install.go

```
@@ -29,111 +42,245 @@ var installCommands = map[string]string{
 	"gentoo": "  sudo emerge superviz\n",
 }
 
-// bufferedWriter wraps a writer with buffering and error tracking.
+// Global pool for buffered writers to reduce allocations
+var bufferedWriterPool = sync.Pool{
```

golangci-lint (1.64.8)

var `bufferedWriterPool` is unused

(unused)